src/Datadog.Trace/Agent/Transports/HttpClientRequest.cs

```diff
@@ -27,6 +31,27 @@ public void AddHeader(string name, string value)
             _request.Headers.Add(name, value);
         }
 
+        public async Task<IApiResponse> PostAsJsonAsync(IEvent events, JsonSerializer serializer)
+        {
+            var ms = new MemoryStream();
+            {
```

Comment:
Is there a reason for this additional scope?

Reply:
removed in 56eff08dc8899e65b62d0b8786e3b281851090b4